Core/COEditingContext.m

```diff
@@ -369,6 +392,11 @@ - (void)undeletePersistentRoot: (COPersistentRoot *)aPersistentRoot
  */
 - (void)updateDeadRelationshipCacheForUndeletedPersistentRoot: (COPersistentRoot *)aPersistentRoot
 {
+	// FIXME:
```

Comment:
I'm not sure, sorry :-(

I am pretty sure I was forced to add that `#If 0` to make some tests pass, so if you re-enable this code you should see some failure hopefully showing why I had to disable it.

If it works to move that code to `-updateCrossPersistentRootReferencesToPersistentRoot:branch:isDeleted: ` feel free.